### PR TITLE
change prefix URL behaviour

### PR DESCRIPTION
**Pull request by sravfeyn**
Changes prefix url behaviour as per https://docs.google.com/document/d/1xckRTOH9TFbUGOClo3uT5PpIlt6T7Fh2bli6Mhu19xo/edit

@czue 
cc @orangejenny

### CODE REVIEW

**Comment by sravfeyn** [comment]
Needs https://github.com/dimagi/commcare-hq/pull/10098

File: views.py [hunk 1]
```diff
@@ -79,7 +87,8 @@ def i18n_context(self):
 
         return {
             'language_options': localized_options,
-            'current_lang_name': _(aliased_language_name(translation.get_language()))
+            'current_lang_name': _(aliased_language_name(translation.get_language())),
+            'url_uses_prefix': bool(self.kwargs.get('lang_code', False))
```
Comment (czue) [inline views.py]: nit but this might be clearer if it was named `url_has_language_prefix`

**Comment by czue** [comment]
these look good to me. not merging due to the label but no blockers on either of them on my side

**Comment by sravfeyn** [comment]
Will remove no-merge label after https://github.com/dimagi/commcare-hq/pull/10098 gets merged

**Comment by sravfeyn** [comment]
G2M

**Pull request merged**